config/initializers/okcomputer.rb

```diff
@@ -9,10 +9,24 @@ def check
   end
 end
 
+class SidekiqRetriesCheck < OkComputer::Check
+  def check
+    retries_queue_length = Sidekiq::RetrySet.new.size
+    queue_length_text = " (#{retries_queue_length})"
+    if retries_queue_length > 10
+      mark_failure
+      mark_message 'Sidekiq pending retries depth is high; suggests high error rate' + queue_length_text
```

Comment:
Is it a bit clearer to have the number next to what it means?

```suggestion
      mark_message "Sidekiq pending retries depth is high #{queue_length_text}. Suggests high error rate"
```